package.json

```diff
@@ -2,42 +2,46 @@
   "name": "htmlhint",
   "version": "0.16.2",
   "description": "The Static Code Analysis Tool for your HTML",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/htmlhint/HTMLHint.git"
-  },
-  "bugs": {
-    "url": "https://github.com/htmlhint/HTMLHint/issues"
-  },
   "keywords": [
     "html",
     "hint",
     "analysis",
     "javascript"
   ],
   "homepage": "https://htmlhint.com",
+  "bugs": {
+    "url": "https://github.com/htmlhint/HTMLHint/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/htmlhint/HTMLHint.git"
+  },
   "license": "MIT",
+  "main": "dist/htmlhint.js",
+  "module": "dist/core/core.js",
   "bin": {
     "htmlhint": "./bin/htmlhint"
   },
-  "main": "dist/htmlhint.js",
-  "module": "dist/core/core.js",
+  "files": [
+    "bin",
+    "dist"
+  ],
   "scripts": {
-    "commit": "npx git-cz",
-    "prettier": "prettier --write .",
-    "test": "mocha --recursive \"./test/**/*.spec.js\"",
     "build": "node build.js",
-    "build:rollup": "npm run build:min && npm run build:unmin",
     "build:min": "rollup -c --environment NODE_ENV:production --file dist/htmlhint.min.js",
+    "build:rollup": "npm run build:min && npm run build:unmin",
     "build:unmin": "rollup -c",
+    "commit": "npx git-cz",
     "lint": "prettier -c . && eslint . --ext .js,.ts --max-warnings 0",
     "lint:fix": "npm run prettier && npm run lint -- --fix",
+    "prettier": "prettier --write .",
+    "test": "mocha --recursive \"./test/**/*.spec.js\"",
     "test:coverage": "nyc npm run test"
   },
   "husky": {
```

Comment:
sorry, I landed the Husky commit, so you might need to delete this node now

Reply:
that's ok, we can keep these.